config.default

```diff
@@ -304,6 +305,35 @@ regionIndicesToPlot = [6]
 # window)
 movingAveragePoints = 12
 
+[streamfunctionMOC]
+## options related to plotting the streamfunction of the meridional overturning 
+## circulation (MOC)
+
+# Region names for basin MOC calculation.
+# Supported options are Atlantic and IndoPacific
+regionNames = ['Atlantic']
+
+# Mask file for post-processing regional MOC computation
+regionMaskFiles = /path/to/MOCregional/mapping/file
+
+# Size of latitude bins over which MOC streamfunction is integrated
+latBinSizeGlobal = 1.
+latBinSizeAtlantic = 0.5
+latBinSizeIndoPacific = 0.5
+
+# colormap for model results
+colormapName = RdYlBu_r
+# colormap indices for contour color
+colormapIndices = [0, 40, 80, 110, 140, 170, 200, 230, 255]
+# colorbar levels/values for contour boundaries
+colorbarLevelsGlobal = [-25, -15, -10, -5, 0, 5, 10, 20, 30, 40]
+colorbarLevelsAtlantic = [-10, -5, -2, 0, 5, 8, 10, 14, 18, 22]
+colorbarLevelsIndoPacific = [-10, -5, -2, 0, 5, 8, 10, 14, 18, 22]
+# contour line levels
+contourLevelsGlobal = np.arange(-25.1,35.1,10)
+contourLevelsAtlantic = np.arange(-8,20.1,2)
+contourLevelsIndoPacific = np.arange(-8,20.1,2)
+
 [timeSeriesMOC]
```

Comment:
I am thinking to remove this section `timeSeriesMOC` all together. Time series are computed within this same script, so it doesn't seem to make sense to have two separate sections.